### PR TITLE
Add Spanish translation for i18n

### PR DESCRIPTION
**Pull request by picandocodigo**
I have a Rails project in production using rails_admin and I was considering removing it and writing an admin dashboard from scratch. But administrate came to the rescue and provided the right balance between customization and automatic code generation I was needing in this project.

Thank you! Here's my first small contribution to the project, hopefully won't be my last.

### CODE REVIEW

File: administrate/config/locales/administrate.es.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+en:
```
Comment (mgrachev) [inline administrate/config/locales/administrate.es.yml]: Maybe `es`?

Comment (picandocodigo) [inline administrate/config/locales/administrate.es.yml]: Oops, missed that, fixed!

**Comment by graysonwright** [comment]
Thanks!

Anyone from the community want to verify these? My Spanish is a bit rusty.

**Comment by amatriain** [comment]
Spaniard here. Looks good to me.

**Comment by pusewicz** [comment]
:+1:

**Comment by graysonwright** [comment]
Great, thanks!

We're holding off on merging PRs for the moment - we're still rearranging the repo to make it easier to deploy & work with. I'll mark this as ready to go for when we start merging again.

**Comment by graysonwright** [comment]
Merged in as https://github.com/thoughtbot/administrate/commit/d0c01d92f8c28a4240ba1e75291f0172cf04f507. Thanks!

**Pull request closed without merging**